# customized position message
geometry_msgs/Point[] Pose_XYZ
uint16 centroid_pixel_x
uint16 centroid_pixel_y


